lib/message.rb

```diff
@@ -0,0 +1,14 @@
+require 'pg'
+
+class Message
+  def self.all
+    if ENV['ENIRONMENT'] == 'test'
```

Comment:
Typo in 'ENIRONMENT' - should be ENVIRONMENT

Reply:
Great spot! I'll change that now.